http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -2550,9 +2550,7 @@ components:
           schema:
             $ref: "#/components/schemas/CreateAppRequest"
           example:
-            diskConfig:
```

Comment:
I think we're reusing the same example for both v1 and v2 api...you'll need to create a separate model for v2 api if you'd like to update the example